Importer/EventListener/InterpreterExceptionSubscriber.php

```diff
@@ -50,6 +51,8 @@ private function resolveInterpreterExceptionMessage(InterpreterException $except
             return "Row is expected to contain {$exception->getExpectedSize()}, found to have {$exception->getFoundSize()} fields instead.";
         } elseif ($exception instanceof InvalidRowException) {
             return $this->buildInvalidRowExceptionMessage($exception);
+        } elseif ($exception instanceof BooleanFormatException) {
```

Comment:
Maybe it's better to have one parent exception class for all SomethingFormatException classes and to handle it by:

```
} elseif ($exception instanceof WrongValueFormatException) {
    return $exception->getMessage();
}
```

This way we don't have to add new `elseif` every time we add new exception for invalid format.